travis/setup_dependencies_common.sh

```diff
@@ -120,6 +120,13 @@ if [[ $NUMPY_VERSION == dev* ]]; then
 elif [[ $NUMPY_VERSION == stable ]]; then
     conda install $QUIET numpy
     export CONDA_INSTALL="conda install $QUIET python=$PYTHON_VERSION"
+elif [[ $NUMPY_VERSION == pre* ]]; then
+    conda install $QUIET numpy
+    export CONDA_INSTALL="conda install $QUIET python=$PYTHON_VERSION"
+    if [[ -z $(pip list -o --pre | grep numpy | \
+            grep -E "[0-9]rc[0-9]|[0-9][ab][0-9]") ]]; then
+        exit
+    fi
```

Comment:
Ah of course - could you add a comment to that effect? This part might be a pain to replicate on Windows, depending on powershell syntax.